packages/kbn-esql-utils/src/utils/esql_fields_utils.ts

```diff
@@ -40,22 +41,28 @@ export const isESQLColumnSortable = (column: DatatableColumn): boolean => {
   return true;
 };
 
+// Helper function to check if a field is groupable based on its type and esType
+const isGroupable = (type: string | undefined, esType: string | undefined): boolean => {
+  if (type === UNKNOWN_FIELD) {
```

Comment:
Can you keep the comments from the function you changed? I had some comments above each if and I want to keep them